.github/workflows/push-translations.yml

```
@@ -0,0 +1,68 @@
+name: Push Translations to Mojito
+
+on:
+  push:
+    paths:
+      - "client-participation/js/strings/en.js"
+      - "client-admin/src/strings/en.js"
+      # Rebuild on change to workflow itself.
+      - ".github/workflows/push-translations.yml"
+    branches:
+      - 320-continuous-localization
+
+jobs:
+  push-translations:
+    runs-on: ubuntu-latest
+    env:
+      TRANSLATION_BRANCH: 320-continuous-localization
+      CHECKOUT_DIR: code
+      MOJITO_VERSION: "0.110"
+      MOJITO_CLI_SERVER: polis-translations.herokuapp.com
+      MOJITO_CLI_USERNAME: admin
+      MOJITO_CLI_PASSWORD: ${{ secrets.MOJITO_CLI_PASSWORD }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.0
+        with:
+          ref: ${{ env.TRANSLATION_BRANCH }}
+          path: ${{ env.CHECKOUT_DIR }}
+
+      - name: Cache Mojito CLI
+        uses: actions/cache@v2.0.0
+        with:
+          path: mojito-cli.jar
+          key: ${{ env.MOJITO_VERSION }}-cli
+
+      - name: Download Mojito CLI
+        run: |
+          wget --continue --progress=dot:giga --output-document mojito-cli.jar https://github.com/box/mojito/releases/download/v${MOJITO_VERSION}/mojito-cli-${MOJITO_VERSION}.jar
+
+      - name: Write Mojito CLI config
+        run: cp ${CHECKOUT_DIR}/.github/support/mojito-cli/application.properties .
+
+      - name: Ensure translation projects exist
+        env:
+          PROJECTS: client-participation client-admin
+        run: |
+          for p in $PROJECTS; do
+            # Add "en-CA" as a dummy locale, since this is a required argument. We'll override with real locales in next step. 
+            java -jar mojito-cli.jar repo-view --name polis:$p || java -jar mojito-cli.jar repo-create --locales en-CA --name polis:$p
+          done
+
+      # For documentation on locale notation:
+      # https://www.mojito.global/docs/guides/managing-locales/
+      #
+      # For a list of all possible locale options:
+      # https://www.mojito.global/docs/refs/mojito-locales/
+      - name: Update available locales
+        env:
+          LOCALES_PARTICIPATION: '"ar-LB" "pt-BR" "uk-UA" "es-419" "da-DK" "de-DE" "fr-FR" "it-IT" "ja-JP" "nl-NL" "zh-TW" "(zh-CN)"'
+          LOCALES_ADMIN:         '"ar-LB" "pt-BR" "uk-UA"'
```

Adding Hebrew as per https://github.com/pol-is/polis/issues/513#issuecomment-671976146

```suggestion
          LOCALES_PARTICIPATION: '"ar-LB" "he-IL" "pt-BR" "uk-UA" "es-419" "da-DK" "de-DE" "fr-FR" "it-IT" "ja-JP" "nl-NL" "zh-TW" "(zh-CN)"'
          LOCALES_ADMIN:         '"ar-LB" "he-IL" "pt-BR" "uk-UA"'
```

cc: @avirankatzsofi

---

Merging as per Aviran's interest in https://github.com/pol-is/polis/issues/513#issuecomment-672968282